lte/gateway/c/oai/common/common_defs.h

```diff
@@ -105,15 +106,18 @@ typedef enum {
   sIZE += sizeof(uint8_t)
 
 #define DECODE_U16(bUFFER, vALUE, sIZE)                                        \
-  vALUE = ntohs(*(uint16_t*) (bUFFER));                                        \
+  memcpy((unsigned char*) &vALUE, bUFFER, sizeof(uint16_t));                   \
```

Comment:
Same question as above - do we *expect* that these buffers might be unaligned? If so then no surprises here and great. In this particular fixup I could see it being less surprising - if this is say parsing a uint16_t that came after a type field that was a byte or whatever.

Reply:
Buffers are mainly packed data (IEs) received over the wire. Thus, no guarantee on alignment